### PR TITLE
gatsby features page

### PR DESCRIPTION
**Pull request by calcsam**
Add v0 of the new features page to Gatsby website. Preview at https://oceanographer-panda-78512.netlify.com/features/

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit c68fb098c4cce9d2ff06f5f662a3bc6fbd821b74

https://app.netlify.com/sites/using-glamor/deploys/59e09a830b79b72c7ebe4a09

File: www/gatsby-browser.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+exports.onRouteUpdate = (location) => {
+  const hash = location.hash || (location.location && location.location.hash)
```
Comment (KyleAMathews) [inline www/gatsby-browser.js]: This is already setup in gatsby-link — try debugging that to see why it's not working? https://github.com/gatsbyjs/gatsby/blob/22e8755b1c0b178197cfeb7fa17ebf9f8839b4ac/packages/gatsby-link/src/index.js#L126

File: www/src/components/container.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ export default ({ children, className, hasSideBar = true }) => (
       [presets.Tablet]: {
         paddingBottom: rhythm(1.5),
       },
+      ...css,
```
Comment (KyleAMathews) [inline www/src/components/container.js]: This isn't necessary as the Glamor plugin we add converts the `css` prop into a `className` before passing it in which is already being added to the div.

File: www/src/components/evaluation-cell.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        }
+        case ``:
+        case `N/A`: {
+          return `none`
+        }
+      }
+      return `none`
+    }
+    const basicStyling = {
+      width: `30px`,
```
Comment (KyleAMathews) [inline www/src/components/evaluation-cell.js]: Please replace all pixel values with usages of rhythm. This helps keep spacing consistent across the site.

File: www/src/components/sidebar-body.js [hunk 4]
```diff
@@ -105,9 +105,15 @@ class SidebarBody extends React.Component {
                     key={section.links[title]}
                     css={{ marginBottom: options.blockMarginBottom / 2 }}
                   >
-                    <Link to={section.links[title]} css={linkStyle}>
-                      {changedTitle}
-                    </Link>
+                    {
+                      section.links[title][0] == `#` ?
+                        <a href={section.links[title]} css={linkStyle}>
```
Comment (KyleAMathews) [inline www/src/components/sidebar-body.js]: oh maybe this is why clicking the links aren't scrolling? I think just using `<Link>` here would solve it.

**Comment by KyleAMathews** [comment]
https://deploy-preview-2443--gatsbyjs.netlify.com/features/

**Comment by fk** [comment]
This is great! 👍 

Two things I'd like to add to Kyle's thoughts (disclaimer: I'm 🤒🤕):

1. The table content currently runs wider than the layout container: bildschirmfoto 2017-10-13 um 18 18 45  
While the aforementioned is remedied relatively easy – we could for example win some horizontal space by turning the "Category" column into its own row – we also need to make the tables behave on mobile/small screens:  bildschirmfoto 2017-10-13 um 18 21 19  
An easy way is to scroll the whole table like Bootstrap does: https://getbootstrap.com/docs/4.0/content/tables/#responsive-tables … and then there's  
  * http://filamentgroup.github.io/tablesaw/demo/kitchensink.html
  * https://zurb.com/playground/projects/responsive-tables/index.html
  * https://hashnode.com/post/really-responsive-tables-using-css3-flexbox-cijzbxd8n00pwvm53sl4l42cx and demo: https://codepen.io/vasansr/full/PZOJXr
  * https://ua-oira.github.io/react-super-responsive-table/
2. I think we should indicate that the "Feature" links are no regular links but toggle additional information. Adding a caret or info icon would do IMO?

Happy to help!

**Comment by calcsam** [comment]
Cool 👍 , doing a couple rounds of feedback incorporating these changes and then will send back

**Comment by calcsam** [comment]
@fk -- really helpful thoughts on making the content responsive, ended up going with a similar solution to the ones you suggested. Still needs a few tweaks but almost good to go.

**Comment by calcsam** [comment]
@KyleAMathews okay we're good for another look on this. Site is now responsive and margins and alignments behave reasonably.

**Comment by KyleAMathews** [comment]
Sweet! Tons of improvement since last time I looked. A few thoughts:

* I miss the lines in between rows. They blend a bit too much now visually. Why not just keep the default table styles there? You can see a table at the bottom of https://www.gatsbyjs.org/packages/gatsby-image/
* highlighting of links is weird in a few places e.g. "Component ecosystem"
* Spacing of columns varies quite a bit. Those should be equal

I notice a few copy tweaks I want to suggest. Monday morning I'll print out the page and ink it up with suggestions.

**Comment by KyleAMathews** [comment]
Though if you're itching to set it live, the copy editing can wait until after it's deployed.

**Comment by calcsam** [comment]
Cool, I'll

**Comment by KyleAMathews** [comment]
I dunno that the hover effect is necessary here so perhaps it'd be better to just kill it rather than have it look silly sometimes.

**Comment by calcsam** [comment]
Okay, got the hover effect working, added the bottom border back, equally spaced the columns. We're good to go.

**Comment by KyleAMathews** [comment]
Looking great! One last visual tweak as I was scrolling through would be to reduce the font-size of labels on each row as they'd fit better smaller and also for info hierarchy that'd be better as they'd be smaller than the font at the top of each section.

**Comment by calcsam** [comment]
Done 👍

**Comment by KyleAMathews** [comment]
🎉  

Fantastic addition to the website!

:shipit:

**Comment by tjgilpin** [comment]
@KyleAMathews @calcsam - Noticed a typo in the last sentence of the second list item for CMSs in the Features section

_Depending onYou can self-host your website_, or use an official hosting provider.

should probably read something like

_Depending on your requirements you can self-host your website_, or use an official hosting provider ?

**Comment by KyleAMathews** [comment]
Would love a PR fixing things!

**Pull request merged**